xml/s4s_setup_cluster.xml

```diff
@@ -674,6 +681,7 @@
    <step>
     <para>
      Save the configuration on its last screen.
+     <remark>sknorr 2019-05-29: how?</remark>
```

Comment:
In the last screen there is a button to save the configuration.

Reply:
I was hoping toms could include the actual button name here. :)
I think it's just called "Save configuration". (yast2-sap-ha-1.0.5/src/lib/sap_ha/wizard/summary_page.rb:49).

Reply:
Correct!

Reply:
Perfect, I've corrected this sentence to "On the last screen, click the `Save configuration` button."